static/app/utils/theme/useChonkPrompt.spec.tsx

```diff
@@ -0,0 +1,117 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {renderHook, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import type {Organization} from 'sentry/types/organization';
+import {QueryClient, QueryClientProvider} from 'sentry/utils/queryClient';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+
+import {useChonkPrompt} from './useChonkPrompt';
+
+const queryClient = new QueryClient();
```

Comment:
note: this will share a `queryClient` between tests. I think it’s better to create a new one for each test in `makeWrapper`.

Reply:
Uf, yes. Oversight on my end, definitely not desirable to reuse these